src/components/Sidebar/Tabs/CustomizationTab/MarkSubTab.tsx

```diff
@@ -0,0 +1,140 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/react';
+import { useState } from 'react';
+
+import {
+  vegaCategoricalChartList,
+  vegaChartList,
+  vegaTemporalChartList,
+} from '../../../../modules/VegaEncodings';
+import SearchBar from '../../../ui-widgets/SearchBar';
+import { CustomizeSubTapProps } from './CustomizationTab';
+import { GraphSpec, useModelState } from '../../../../hooks/bifrost-model';
+import { PlainObject, VegaLite, VisualizationSpec } from 'react-vega';
+import produce from 'immer';
+import theme from '../../../../theme';
+
+const markOptionsListCss = css`
+  display: flex;
+  flex-wrap: wrap;
+  overflow: scroll;
+  list-style: none;
+  padding: 0px;
+`;
+
+const markOptionCss = css`
+  display: flex;
+  flex-direction: column;
+
+  &.selected {
+    border: 5px solid ${theme.color.primary[2]};
```

Comment:
General thoughts on the color system? The arrays are kinda handy, but could also cause issues if we delete an earlier element.

Reply:
maybe I can create `onSelected` field in theme and use that? Something like below?
`const theme = {
  color: {
    text: ['#000'],
    background: ['#fff', '#F6F6F6'],
    primary: ['#771C79', '#AD77AF'],
    onSelected: '#E4D2E4'
  },
};`

Reply:
Idk will that color be true to all things selected in the entire app?

Reply:
Indexing into primary makes sense, we could also do something like

```

const theme = { 
color: {
 primary: {
       light: '#771C79',
       standard: '#AD77AF',
       dark: "#..."
        } 
... }
 };
```
As an option. The one advantage of this is that the color tags are consistent regardless of how the dict gets manipulated.

Reply:
Agree. Will just go with that for now